docs/proposals/#52-i18n(Chinese)/proposal.md

```diff
@@ -0,0 +1,199 @@
+## Project Background: Current State & Pain Points
```

Comment:
Add a title for this proposal.  like `UI Optimization and i18n (Chinese) Support for Dashboard`